api/models/storeDao.js

```diff
@@ -0,0 +1,86 @@
+const appDataSource = require('./dataSource')
+
+const getAllStores = async () => {
+  //try{
+    return await appDataSource.query(
```

Comment:
```suggestion
    return appDataSource.query(
```
동기적으로 수행되어야할 다음 줄의 코드(로직)가 없기 때문에 await는 필요없습니다!